tests/migrate/test_context_real_examples.py

```diff
@@ -142,67 +142,63 @@ def setUp(self):
             ),
             FTL.Message(
                 id=FTL.Identifier('delete-all-title'),
-                value=LITERAL_FROM(
+                value=COPY(
                     'aboutDownloads.properties',
                     'downloadAction.deleteAll'
                 )
             ),
             FTL.Message(
                 id=FTL.Identifier('delete-all-message'),
```

Comment:
This message is a good example of how the renaming makes writing migrations easier.